pio/ws2812/ws2812.c

```diff
@@ -12,6 +12,10 @@
 #include "hardware/clocks.h"
 #include "ws2812.pio.h"
 
+#define PIN_TX 2
+#define NUM_PIXELS 150
+#define IS_WRGB true
```

Comment:
Hmmm, a quick search suggests that `RGBW` is more common nomenclature than `WRGB`?

Reply:
Yes, you are right. Will change to RGBW.